ami/logger.py

```diff
@@ -0,0 +1,30 @@
+import logging
+import logging.config
+
+import yaml
+
+
+def get_main_thread_logger(name: str) -> logging.Logger:
+    return __get_thread_logger("main", name)
+
+
+def get_trainer_thread_logger(name: str) -> logging.Logger:
+    return __get_thread_logger("trainer", name)
```

Comment:
学習スレッドの名前は`training`で統一します。
```suggestion
def get_training_thread_logger(name: str) -> logging.Logger:
    return __get_thread_logger("training", name)
```

Reply:
推論スレッドは `InferenceThread` ですか？ `InteractionThread` ですか？